lib/ploomes_client/deal/list.rb

```diff
@@ -12,7 +12,21 @@ def make_request
       end
 
       def path_template
-        '/Deals?$expand=Attachments,OtherProperties,Pipeline,Status,LossReason&$filter=Id+eq+%<id>s'
+        "/Deals?$expand=#{expanded_fields}&$filter=Id+eq+%<id>s"
+      end
+
+      def expanded_fields
+        # if needed, there is the possibility of passing 'sub-selections' and 'sub-expansions'
```

Comment:
Maybe, as an improvement, we could transform this in a client paramenter, with these default versions.

Reply:
I think that this is going to be a TODO since the querying is complex and it would take time to develop a more agnostic solution to the problem. It is a good idea